jobs/update-colin-filings/Jenkinsfiles/run-cu-prod.groovy

```diff
@@ -0,0 +1,44 @@
+// Copyright © 2018 Province of British Columbia
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+//JENKINS DEPLOY ENVIRONMENT VARIABLES:
+// - JENKINS_JAVA_OVERRIDES  -Dhudson.model.DirectoryBrowserSupport.CSP= -Duser.timezone=America/Vancouver
+//   -> user.timezone : set the local timezone so logfiles report correxct time
+//   -> hudson.model.DirectoryBrowserSupport.CSP : removes restrictions on CSS file load, thus html pages of test reports are displayed pretty
+//   See: https://docs.openshift.com/container-platform/3.9/using_images/other_images/jenkins.html for a complete list of JENKINS env vars
+
+// define constants
+NAMESPACE = 'gl2uos'
+TAG_NAME = 'prod'
```

Comment:
Wait, this runs against prod?

Reply:
Or is this the normal COLIN Updater job?

Reply:
All this does is run the job (it doesn't test anyhting) so we might want it for ops. It doesn't have any negative consequences

Reply:
these jobs are already running regularly. All this does is run it right away so we don't have to wait for it